### PR TITLE
[Multi Bridge] Display Price Impact and Expected Output

### PR DESCRIPTION
**Pull request by JoseRFelix**
## What is the purpose of the change

Display price impact and expected output for Deposits and Withdraws. If the output is too low (6% less than the input) or if the price impact is too high (10%), display the price in red and change button text to "Transfer Anyway" to bring the user's attention.

### ClickUp Task

[ClickUp Task URL](https://app.clickup.com/t/86a1duk5p)

### Pending task

- [ ] Add disclaimer link

## Brief Changelog

image

image

image

image

## Testing and Verifying

- [ ] Should display a warning if the expected output is too low or if the price impact is too high.

### CODE REVIEW

**Comment by kamal-sutra** [comment]
Shouldn't Expected Output deduct Transfer fee and display final amount? I see the warning there, but users wont notice it. imo, it's better to display lower output deducting fees along with the warning. For better conveying, display the warning upfront (under the expected output) instead of 'i'

image
image

**Comment by kamal-sutra** [comment]
For Squid- Expected output doesn't deduct fee.
For Axelar- It deducts fee

image
image

**Comment by kamal-sutra** [comment]
No Quotes for Aval.USDC, Poly.usdc, xc.Dot

image
image
image

**Comment by JoseRFelix** [comment]
> Shouldn't Expected Output deduct Transfer fee and display final amount? I see the warning there, but users wont notice it. imo, it's better to display lower output deducting fees along with the warning. For better conveying, display the warning upfront (under the expected output) instead of 'i'
> 
> image image

I believe this is working as expected given Squid like Axelar does deduct transfer fees but not gas fees. In this case, the transfer fees are so low that they do not affect the expected output. Furthermore, the fees are much lower compared to Axelar because they are not using Axelar bridge transfer but General Message Passing. We can confirm this by looking for a route that goes through Axelar on Squid like ETH. The values are identical.

**Comment by JoseRFelix** [comment]
> No Quotes for Aval.USDC, Poly.usdc, xc.Dot
> 
> image image image

Hmm, these are working for me. Can you try again 🙏

**Review by kamal-sutra — APPROVED** [review]

**Comment by CryptoAssassin1** [comment]
@JoseRFelix ETH Assets Deposit/Withdraw page content is overlapping on my laptop resolution 1360x768 on main app its fine.
Screenshot from 2023-11-15 16-00-53
Screenshot from 2023-11-15 16-01-09
Screenshot from 2023-11-15 16-02-19

**Review by jonator — APPROVED** [review]

File: packages/utils/src/common-utils.ts
[the remark this answers is not in the document]
Comment (JoseRFelix) [reply packages/utils/src/common-utils.ts:1]: This util is used to evaluate if it's not defined. Boolean considers `0` to be `false` even though it's defined.

**Pull request merged**